docs/how-to/installation/aws.txt

```diff
@@ -221,11 +206,11 @@ Master
 ^^^^^^
 - Egress on all ports to agent security group
 
-- Egress outbound to the internet
+- Egress outbound to the Internet
 
-- Ingress on 8080 to view to the Determined UI
+- Ingress on port 8080 to view to the Determined WebUI
```

Comment:
blocking: I think we should stop referring to the front-end as the "WebUI", it feels like a term that's very specific to our internal usage. "UI" or "web interface" feels more general and accessible.

Reply:
in other words...

drop the Web, just UI. it's cleaner